lib/less/rails/railtie.rb

```diff
@@ -14,11 +14,16 @@ class Railtie < ::Rails::Railtie
         require 'less-rails'
         Sprockets::Engines #force autoloading
 
-        if Sprockets.respond_to?(:register_engine)
-          args = ['.less', LessTemplate]
-          args << { mime_type: 'text/less', silence_deprecation: true } if Sprockets::VERSION.start_with?("3")
-          env.register_engine(*args)
-        end
+        config.assets.configure do |env|
+          if env.respond_to?(:register_engine)
+            args = ['.less', LessTemplate]
+            args << { mime_type: 'text/less', silence_deprecation: true } if env::VERSION.start_with?("3")
```

Comment:
This line is broke tests:

```
/home/travis/build/metaskills/less-rails/lib/less/rails/railtie.rb:20:in `block (2 levels) in <class:Railtie>': #<Sprockets::Environment:0x2077c84 root="/home/travis/build/metaskills/less-rails/test/dummy_app", paths=["/home/travis/build/metaskills/less-rails/test/dummy_app/app/assets/images", "/home/travis/build/metaskills/less-rails/test/dummy_app/app/assets/stylesheets", "/home/travis/build/metaskills/less-rails/test/dummy_app/vendor/assets/stylesheets"]> is not a class/module (TypeError)
```

Why are you start using `env::VERSION` instead of `Sprockets::VERSION`?